autogpt_platform/frontend/src/app/page.tsx

```diff
@@ -15,17 +16,51 @@ import {
   FlowRunsList,
   FlowRunsStats,
 } from "@/components/monitor";
+import { SchedulesTable } from "@/components/monitor/scheduleTable";
 
 const Monitor = () => {
   const [flows, setFlows] = useState<GraphMetaWithRuns[]>([]);
   const [flowRuns, setFlowRuns] = useState<FlowRun[]>([]);
+  const [schedules, setSchedules] = useState<Schedule[]>([]);
   const [selectedFlow, setSelectedFlow] = useState<GraphMetaWithRuns | null>(
     null,
   );
   const [selectedRun, setSelectedRun] = useState<FlowRun | null>(null);
+  const [sortColumn, setSortColumn] = useState<keyof Schedule>("id");
+  const [sortDirection, setSortDirection] = useState<"asc" | "desc">("asc");
 
   const api = useMemo(() => new AutoGPTServerAPI(), []);
 
+  const fetchSchedules = useCallback(async () => {
+    const schedulesData: Schedule[] = [];
+    for (const flow of flows) {
```

Comment:
I see that you did this due to non-optimized backend, I've improved the API here so you can do GET /schedules and get all the schedules for a user without firing multiple requests and add more information on the GET response:
https://github.com/Significant-Gravitas/AutoGPT/pull/8649

I'm happy with any of the ordering, we can first get this merged and improve it and update the routing on my PR, or you can rebase your PR based on the above PR.